guides/common/modules/proc_adding-errata-to-a-host.adoc

```diff
@@ -0,0 +1,14 @@
+[id="Adding_errata_to_a_single_host_{context}"]
+= Adding errata to a single host
+
+Use the following procedure to add errata to a host.
+
+.Procedure
+. In the Satellite web UI, navigate to *Hosts > All Hosts* and select the host you want to add Errata to.
+. On the new Host page, select the *Content* tab.
+. On the *Content* page select the *Errata* tab.
+. Select the errata you want to add to the host or select the checkbox at the top of the list to add all installable errata.
+Click the check box next to any errata you wish to remove from a full list.
+. Using the three dots icon of the errata you want to add to the host, select *Apply via Katello agent* if you have no connectivity to the target host using SSH.
```

Comment:
IMHO we should not recommend users using "Katello Agent" any longer.
Maybe change the order and list REX first? Maybe add `(Deprecated)` to it? Or maybe even leave it out for now, because this is docs for Foreman 3.2+.